plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -513,6 +462,10 @@ fu_plugin_uefi_capsule_coldplug_device (FuPlugin *plugin, FuUefiDevice *dev, GEr
 		fu_device_add_private_flag (FU_DEVICE (dev),
 					    FU_UEFI_DEVICE_FLAG_USE_LEGACY_BOOTMGR_DESC);
 	}
+	if (fu_plugin_has_custom_flag (plugin, "boot-order-lock")) {
```

Comment:
and shouldn't it be using the private flag not the custom flag?

Reply:
**fu_plugin** - not fu_device